src/protocols/core/Seat.cpp

```diff
@@ -451,12 +452,20 @@ void CWLSeatProtocol::updateCapabilities(uint32_t caps) {
 }
 
 void CWLSeatProtocol::updateKeymap() {
+    if (!(currentCaps & eHIDCapabilityType::HID_INPUT_CAPABILITY_KEYBOARD)) {
```

Comment:
no {}

Reply:
Is 6bd18af3d6ec43b3efac142f410b7d20132f7893 OK?